### PR TITLE
#4987 [Manager] New added plugins not showing in toolbox after restart workspace

### PR DESCRIPTION
**Pull request by saivirtue**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4987)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/common/Progress/LogProgress.js [hunk 1]
```diff
@@ -165,23 +165,21 @@ const LogProgress = props => {
           })}
         </Stepper>
       )}
-      <div className={'FlexDiv'} isHide={isHide}>
+      <div className={'FlexDiv'}>
```
Comment (saivirtue) [inline ohara-manager/client/src/components/common/Progress/LogProgress.js:168]: We don't need this custom props passing through each individual HTML element now :)

File: ohara-manager/client/src/components/common/Progress/LogPropessStyles.js [hunk 2]
```diff
@@ -14,10 +14,16 @@
...
+import React from 'react';
 import styled, { css } from 'styled-components';
 import Dialog from '@material-ui/core/Dialog';
 
-const StyledDialog = styled(Dialog)(
+const StyledDialog = styled(props => (
+  // Don't pass down props that are meant only used by styled-component.
+  // React also complains about this if these props are not omitted
+  <Dialog {...omit(props, ['isHide', 'isDeleteDialog'])} />
+))(
```
Comment (saivirtue) [inline ohara-manager/client/src/components/common/Progress/LogPropessStyles.js:26]: Big thanks @konekoya ! 🙇‍♂️

File: ohara-manager/client/src/components/common/Progress/LogPropessStyles.js [hunk 3]
```diff
@@ -44,31 +50,31 @@ const StyledDialog = styled(Dialog)(
     }
     .RightFlexDiv {
       display: flex;
-      flex:${isHide ? 0 : 1}
+      flex: ${isHide ? 0 : 1};
```
Comment (konekoya) [inline ohara-manager/client/src/components/common/Progress/LogPropessStyles.js:53]: Sorry Sam, did mention this in previous PR reviews. This should probably be `isHidden` :)

File: ohara-manager/client/src/store/epics/workspace/restartWorkspaceEpic.js [hunk 4]
```diff
@@ -399,16 +399,18 @@ const waitStopTopic$ = params =>
...
     of(
       actions.createEventLog.trigger({
-        title: `Successfully Restart workspace ${params.name}.`,
+        title: `Successfully Restart workspace ${workspaceKey.name}.`,
         type: LOG_LEVEL.info,
       }),
     ),
     // Refetch node list after creation successfully in order to get the runtime data
     of(actions.fetchNodes.trigger()),
+    // Refetch connector list (inspect worker and shabondi) for those new added plugins
```
Comment (konekoya) [inline ohara-manager/client/src/store/epics/workspace/restartWorkspaceEpic.js:412]: 👍

**Comment by saivirtue** [comment]
retry manager-e2e

**Comment by saivirtue** [comment]
retry manager-e2e

**Pull request merged**